game-of-life-multiplatform/iOS/src/main/kotlin/com/novoda/gol/GameLoop.kt

```diff
@@ -0,0 +1,25 @@
+package com.novoda.gol
+
+import platform.Foundation.*
+
+actual class GameLoopImpl : GameLoop {
```

Comment:
This has been changed compared to the native client where we implemented the [GameLoop](https://github.com/novoda/spikes/blob/master/game-of-life-native/Game%20of%20Life/SwiftGameLoop.swift) in swift.

In this case we implement it as `actual` kotlin class based on [NSTimer](https://developer.apple.com/documentation/foundation/nstimer).